api/envoy/extensions/filters/http/dynamic_forward_proxy/v3/dynamic_forward_proxy.proto

```diff
@@ -41,6 +41,14 @@ message FilterConfig {
   // ``envoy.stream.upstream_address`` (See
   // :repo:`upstream_address.h<source/common/stream_info/upstream_address.h>`).
   bool save_upstream_address = 2;
+
+  // When this flag is set, the filter will check for the "envoy.upstream.dynamic_host" and/or
+  // "envoy.upstream.dynamic_port" filter state values before using the HTTP Host
+  // header for DNS resolution. This provides consistency with the SNI and UDP
+  // Dynamic Forward Proxy filters behavior when enabled.
```

Comment:
Please add links to the SNI and UDP dynamic forward proxy features (assuming there are corresponding fields/descriptions of the behavior).